code/modules/client/preference_setup/loadout/loadout_uniform.dm

```diff
@@ -450,84 +390,24 @@
 	display_name = "sweater, retro"
 	path = /obj/item/clothing/under/retrosweater
 
-/datum/gear/uniform/brandsuit/aether
-	display_name = "jumpsuit, aether"
+/datum/gear/uniform/brandsuit
+	display_name = "jumpsuit/uniform, corporate selection"
 	path = /obj/item/clothing/under/corp/aether
 
-/datum/gear/uniform/brandsuit/centauri
-	display_name = "jumpsuit, centauri provisions"
-	path = /obj/item/clothing/under/corp/centauri
-
-/datum/gear/uniform/brandsuit/focal
-	display_name = "jumpsuit, focal point"
-	path = /obj/item/clothing/under/corp/focal
-
-/datum/gear/uniform/brandsuit/grayson
-	display_name = "outfit, grayson"
-	path = /obj/item/clothing/under/corp/grayson
-
-/datum/gear/uniform/brandsuit/grayson_jump
-	display_name = "jumpsuit, grayson"
-	path = /obj/item/clothing/under/corp/grayson
-
-/datum/gear/uniform/brandsuit/hedberg
-	display_name = "uniform, hedberg officer (Security)"
-	path = 	/obj/item/clothing/under/hedberg
-	allowed_roles = list("Security Officer","Head of Security","Warden")
-
-/datum/gear/uniform/brandsuit/hedbergtech
-	display_name = "jumpsuit, hedberg technician"
-	path = 	/obj/item/clothing/under/hedbergtech
```

Comment:
This seems to have disappeared without landing elsewhere, given it's not /under/corp